rdbms/models/addresses.py

```diff
@@ -0,0 +1,17 @@
+from manage import Base
+from sqlalchemy import Column, Integer, String, ForeignKey
+
+
+class Addresses(Base):
+    __tablename__ = 'addresses'
+
+    id = Column(Integer, primary_key=True, nullable=False)
+    user_id = Column(Integer, ForeignKey('users.id'))
+    street_address = Column(String(50), nullable=False)
+    street_address_two = Column(String(50), nullable=False)
+    city = Column(String(20), nullable=False)
+    state = Column(String(15), nullable=False)
+    postal_code = Column(Integer, nullable=False)
+    country = Column(String(20), nullable=False)
+    start_date = Column(Integer, nullable=False)
```

Comment:
Incorrect data type for dates. See the list of available generic data types and select an appropriate data type for dates.
https://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.Date